### PR TITLE
Replace service_account_json (file) with service_account_json_path (string) to allow call-caching

### PR DESCRIPTION
**Pull request by rsasch**
Otherwise, when json is refreshed, contents of the file are different, hash of the file is different, and call-caching will not register a match, despite the same "account" being used.

- changed input type from `File` to `String`
- changed the name to make it more obvious/clear

Closes https://github.com/broadinstitute/dsp-spec-ops/issues/327

### CODE REVIEW

File: scripts/variantstore/wdl/GvsImportGenomes.wdl [hunk 1]
```diff
@@ -201,20 +201,21 @@ task SetLock {
...
     String output_directory
-    File? service_account_json
+    String? service_account_json_path
 
     # runtime
     Int? preemptible_tries
   }
 
-  String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
+  String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsImportGenomes.wdl:210]: why are these a string and not a boolean?

File: scripts/variantstore/wdl/GvsImportGenomes.wdl [hunk 2]
```diff
@@ -632,23 +637,23 @@ task LoadTable {
...
 
   command <<<
     set -x
     set -e
 
     if [ ~{has_service_account_file} = 'true' ]; then
-      gcloud auth activate-service-account --key-file='~{service_account_json}'
-      gcloud config set project ~{project_id}
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsImportGenomes.wdl:651]: why is the project id not being set anymore?

**Review by RoriCremer — APPROVED** [review]
the proj id is the only thing I'm worried about---but maybe I just dont understand

File: scripts/variantstore/wdl/GvsImportGenomes.wdl (hunk 2 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsImportGenomes.wdl:651]: whoops — good catch!  should be back in

File: scripts/variantstore/wdl/GvsImportGenomes.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsImportGenomes.wdl:210]: My guess would be that testing a boolean value in bash that was set in WDL is not particularly straightforward.

**Pull request merged**